deploy/osd-serviceaccounts/wif/00-serviceaccounts.yaml

```diff
@@ -0,0 +1,30 @@
+---
```

Comment:
Can we merge this whole file into the sts folder and call the folder `sts-wif`? If I'm not mistaken, it's the same config.

Reply:
True, it is the same config indeed. Will try to do that if there is a way to do an OR on "sts" and "wif" conditions

Reply:
OR is not a possibility
https://github.com/openshift/hive/blob/master/config/crds/hive.openshift.io_selectorsyncsets.yaml#L64
`matchExpressions` can only be ANDed
Checking if `00-serviceaccounts.yaml` can be defined at a parent level.

Reply:
Ok it does not look it is possible to do this parent level sharing:
```
> for f in `find . | grep '\.yaml$'`; do if ! [ -f "$(dirname "$f")/config.yaml" ]; then echo "$f"; fi; done
./resources/managed/all-osd-resources.yaml
./resources/prometheusrules/fluentd_openshift-logging_collector.PrometheusRule.yaml
./resources/prometheusrules/elasticsearch_openshift-logging_elasticsearch-prometheus-rules.PrometheusRule.yaml
./resources/wif/4.15/vanilla.yaml
./resources/wif/4.14/vanilla.yaml
./resources/wif/4.16/vanilla.yaml
./resources/addons-namespaces/main.yaml
./resources/tailored-profiles/hypershift-cis-tp.yaml
./resources/tailored-profiles/hypershift-pcidss-tp.yaml
./scripts/policy-generator-config.yaml
./scripts/templates/template.yaml
./scripts/templates/selectorsyncset.yaml
```
YAML files in `./deploy` are always with a `config.yaml` file in the same folder.